cmd/mattermost/commands/config.go

```diff
@@ -68,15 +68,27 @@ var ConfigSetCmd = &cobra.Command{
 	RunE:    configSetCmdF,
 }
 
+var MigrateConfigCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrate config file",
+	Long:  "Instantiates a new config store and sets it based on an existing defined store",
+	Example: `	config migrate --from=path/to/config.json --to=a/different/path/to/configuration.json
```

Comment:
I don't think going from a file to a file is a meaningful example: while it should be allowed, it's not illustrative of how this is meant to be used. Propose removing this example.